README.md

```diff
@@ -64,18 +67,25 @@ PublicLab.Module.extend({
 });
 ````
 
+## Installation
+
+To install PublicLab.Editor for development, you'll need [NodeJS](https://nodejs.org). You can get the detailed instruction on installing node and npm in its official [documentation](https://docs.npmjs.com/getting-started/installing-node).
+
+After installing node and npm run `npm install` from the root directory.
+
+PublicLab.Editor uses grunt - the javascript task runner for compilation of the modules. To install grunt run `npm install -g grunt-cli`. You may have to use `sudo` for root privileges.
+
+Make changes to the files in the `/src/` directory, then run `grunt build` to compile into `/dist/PublicLab.Editor.js`. This will use `grunt-browserify` to concatenate and include any node modules named in `require()` statements. You'll then be able to try it out in `/examples/index.html`. Run `grunt` and leave it running to build as you go.
+
 
 ## Setup
 
 To use PublicLab.Editor, you'll need to follow [the template provided here](https://publiclab.github.io/PublicLab.Editor/examples/index.html), and use the following constructor:
 
 ````js
-var editor = new PublicLab.Editor({ 
+var editor = new PL.Editor({ 
   textarea: document.getElementById('my-textarea'),
-  id:                "username", // optional unique id for localStorage history
-  publishUrl:        "/notes",   // content will POST to this URL upon clicking "Publish"
-  updateUrl:         "/notes",   // content will UPDATE to this URL upon clicking "Save"
-  relatedTagsUrl:    "/notes",   // content will UPDATE to this URL upon clicking "Save"
+  destination:        "/notes/create",   // content will Submit to this URL upon clicking "Publish"
```

Comment:
@jywarren I believe this is largely changed in the recent version and these parameters don't exist now. Here is a refernce of the constructor definition https://github.com/publiclab/PublicLab.Editor/blob/master/examples/index.html#L219-L233. I am not sure though how the TagsModule works.